config/settings.d/logs.yml.example

```diff
@@ -0,0 +1,5 @@
+---
+# Can be true, false, or http/https to enable just one of the protocols
+:enabled: false
+
+# Log buffer configuration is in core file (settings.yml)
```

Comment:
If log buffer is only used to show it in the Foreman smart proxy page - can we move the log_buffer and log_buffer errors to this file instead?